### PR TITLE
Update ctod.dd

### PR DESCRIPTION
**Pull request by Kozzi11**
Fix static array declaration

### CODE REVIEW

File: ctod.dd [hunk 1]
```diff
@@ -266,8 +266,7 @@ for (i = 0; i &lt; ARRAY_LENGTH; i++)
 $(H4 The D Way)
 
 ----------------------------
-int array[17];
-array[] = value;
+int[17] array = value;
```
Comment (aG0aep6G) [inline ctod.dd]: I think the assignment should stay separate. The initialization way might be a bit misleading, as it only works with fixed-size arrays.

Comment (Kozzi11) [inline ctod.dd]: Ok, what is better:
int[17] array;
array[] = 7; // or array = 7;

Comment (aG0aep6G) [inline ctod.dd]: > array[] = 7; // or array = 7;

I'm for `array[] = value;`. The other variant only works with fixed-size arrays again.

**Comment by ZombineDev** [comment]
LGTM

**Comment by 9rnsr** [comment]
Looks good, too.

**Pull request merged**